pallets/ddc-staking/src/tests.rs

```diff
@@ -176,3 +178,32 @@ fn staking_should_work() {
 		assert_eq!(DdcStaking::edges(3), None);
 	});
 }
+
+#[test]
+fn cluster_managers_list_can_be_managed_by_governance_only() {
+	ExtBuilder::default().build_and_execute(|| {
+		// Governance can allow an account to become cluster manager.
+		assert_ok!(DdcStaking::allow_cluster_manager(RuntimeOrigin::root(), 1));
```

Comment:
Why `governance` if the origin is root?

Reply:
As far as I understand governance (sudo, referenda) decisions are dispatched from the root origin.
https://github.com/search?q=repo%3Aparitytech%2Fpolkadot-sdk%20frame_system%3A%3ARawOrigin%3A%3ARoot&type=code